packages/module-plugin/index.js

```diff
@@ -7,6 +7,7 @@ const { BasicAuthRequester } = require('./requester/basic');
 const { OAuth2Requester } = require('./requester/oauth-2');
 const { Requester } = require('./requester/requester');
 const { ModuleConstants } = require('./ModuleConstants');
+const { IntegrationMapping } = require('./integration-mapping');
```

Comment:
I'm torn as to whether this should be here in module-plugin, or over in /packages/integrations

leaning towards integrations, since we want to relate them to the integration record